test/modules/paths/int/ChapelNumLocales.chpl

```diff
@@ -7,9 +7,19 @@ module ChapelNumLocales {
 
   extern proc printf(x...);
   printf("In personal NumLocales module\n");
-  
+
   //
   // the number of locales on which to run the program
   //
   config const numLocales = chpl_comm_default_num_locales();
+
+  // this procedure is a helper for warnUnstableInt and warnUnstableBoth
```

Comment:
Typo/grammar `warnUnstableBoth`.

Reply:
I'm not seeing the typo, those are names of tests, is there something wrong with those names?